pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -510,9 +525,19 @@ def create_bam_file_entry(donor_unique_id, analysis_attrib, gnos_analysis):
         bam_file['bam_type'] = 'Unknown'
         bam_file['alignment'] = None
 
+    if bam_file.get('bam_type') == 'Specimen level aligned BAM' or bam_file.get('bam_type') == 'RNA-Seq aligned BAM':
```

Comment:
don't really need this if statement here, it's ok to have the flag for all bam files regardless of types, the value of the flag will be always correct anyway